.github/workflows/ci.yml

```diff
@@ -9,34 +9,48 @@ jobs:
     container:
       image: registry.fedoraproject.org/fedora:37
     steps:
-      - run: dnf install -y git make
-      - uses: actions/checkout@v4
+      - run: dnf install -y git make rpmlint
+      - name: Lint specfiles
+        uses: actions/checkout@v4
         with:
           persist-credentials: false
-      - name: Install dependencies
-        run: |
-          make test-deps
-      - name: Run linters
-        run: |
+      - run: |
           git config --global --add safe.directory '*'
-          make lint
+          rpmlint rpm_spec/*.spec
   build-rpm:
     runs-on: ubuntu-latest
-    container:
-      image: registry.fedoraproject.org/fedora:37
+    env:
+      BUILDER_CACHE_DIR: ${{ github.workspace }}/.qubes-builderv2-cache
```

Comment:
(we aren't caching any of the images now because of how much I had to fight with ci, but we could `docker save` them to this cache dir and look for them for subsequent ci runs in a followup.)